### DISCUSSION TITLE
Github connect error

### DISCUSSION THREAD

**Discussion by sztartup** [community/community#124961] [New to GitHub]
Hi,

I wanted to link my homey (https://homey.app) to github.
1. I installed the Github app (g1.jpg, g2.jpg)
2. I tried to link them (g3.jpg, g4.jpg)
3. after logging into github, I gave the requested permissions (g5.jpg), then after g6.jpg I went back to homey.
4. g7.jpg flashed for about 1 second, then the new device not found appeared (g8.jpg)
5. looking at the app, still g2.jpg is shown, no Github device connected anywhere (flow, ...).
6. github shows the permissions (g9.jpg), which are also notified by email.

I have tried several times, deleting the permissions on Github, deleting and reinstalling the Github app, but in none of the cases has the connection been successful.

What's wrong?

Best,

**Comment by nithinlycanz** [comment]
Actually you can use git for that, have you tried installing it and upload via git from vs code or something.

**Comment by nithinlycanz** [comment]
Firstly just install git and login, then 
you can use 
git init
git add .
git commit -m "message"
git remote add origin "the repo url" //be sure to create a repo in github.com in your profile
git pull
then git push origin main
that's it buddy.

**Comment by sztartup** [comment]
THX, but cannot be used in this case.
